src/test/java/domain/PlayerTest.java

```diff
@@ -0,0 +1,27 @@
+package domain;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class PlayerTest {
+
+    @Test
+    @DisplayName("5글자이면 player 객체가 생성")
+    void noError() {
+        String inputName = "abcde";
+        assertDoesNotThrow(() -> new Player(inputName));
+
+    }
+
+    @Test
+    @DisplayName("6글자 이상 입력될 경우 예외 발생")
+    void errorAboutNameLimit() {
+        String inputName = "abcdef";
+        assertThatThrownBy(() ->
+                new Player(inputName)).isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("[ERROR] 사람 이름은 최대 5글자 입니다.");
```

Comment:
컨벤션이 익숙치않네요! :) 위와 동일하게 가져가보는 것은 어떨까요-!
```suggestion
assertThatThrownBy(() -> new Player(inputName))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessage("[ERROR] 사람 이름은 최대 5글자 입니다.");
```

Reply:
아직 컨벤션이 익숙지 않아서 놓쳤네요ㅠ

앞으로 좀 더 꼼꼼히 챙기겠습니다!

Reply:
안녕하세요 코일!

바쁘실텐데 피드백을 빠르게 남겨주셔서 감사합니다!

그에 반해 제 답변은 너무 늦었군요.. 죄송합니다ㅠ

먼저 Q&A에 대한 제 생각을 말씀드리자면,

- 사다리 라인 객체가 직접 랜덤값을 생성하는 것이 아닌 다른 방법은 어떤 방법을 생각하셨을까요? 로이께서 어떤 생각을 가지고 계신지 궁금합니다-!
    ->  1) 처음에는 외부에서 랜덤값을 생성하여 사다리 라인 객체로 주입하는 방식을 생각해봤습니다. 하지만 다시 생각해보니 진행할 경우 사다리 라인의 상태가 통일 되어 객체가 생성(모두 이어지거나, 모두 끊어지거나)되는 문제가 있었습니다. 
          2) 게다가 라인의 상태(이어지거나 끊어지거나)를 생성하는 것은 라인 객체의 직접적인 책임에 해당한다는 판단이 들어 랜덤 값을 생성하는 주체는 Line객체가 적합하다고 생각합니다.


- 사다리 객체에 생성자로 주입하는 방법 VS 객체 집합을 직접 만드는 방법의 차이에 대해 로이의 의견이 궁금하네요-! 전자, 후자 중 유연한 객체를 작성하려면 어떻게 작성해야할지 로이의 의견을 말씀해주세요-!
    -> 1) 예를 들어, 외부(LadderMaker)에서 List<Line>을 직접 생성하여  Ladder객체의 생성자에 주입하는 방법은 Ladder객체가 직접 Line을 생성하는 것보다 유연한 방식이라고 생각합니다. 
         2) 하지만 Ladder객체와 Line객체는 포함관계인 만큼, Line 객체의 생성 주체는 외부(LadderMaker)가 아닌 Ladder 객체가 옳다고 생각하여, 해당 방식으로 구현해봤습니다. 


기타 세부적으로 코멘트 주신 내용에 대한 수정 사항은 아래에 답글로 달아봤습니다.

꼼꼼한 피드백 감사합니다 코일!

이번주도 파이팅입니다!